nncf/common/insertion_point_graph.py

```diff
@@ -314,3 +354,35 @@ def get_pre_hook_node_key(node_key: str, input_port_id: int = 0) -> str:
     @staticmethod
     def get_post_hook_node_key(node_key: str) -> str:
         return InsertionPointGraph.POST_HOOK_ID_PREFIX + node_key
+
+
+class ConstantNodesFilter:
+    @staticmethod
+    def filter(ip_graph: InsertionPointGraph, quantizable_layer_node_keys: List[str]) -> InsertionPointGraph:
+        """
+        Removes all Constant nodes from InsertionPointGraph, making it inference graph.
+        The traversing starts from the input nodes and nodes with weights.
+
+        :param ip_graph: The original InsertionPointGraph.
+        :param quantizable_layer_node_keys: Keys of the nodes which have weights.
+        :return: InsertionPointGraph without Constant nodes.
+        """
+        input_nodes = ip_graph.get_input_nodes()
+        if not input_nodes:
+            nncf_logger.debug('The Filtration of Constant nodes of InsertionPointGraph is skipped,'
+                              ' because the graph does not have Input nodes,'
+                              ' from which the traversing should be started.')
```

Comment:
In our tests we have such graphs, which was the main reason of adding this skipping. After the discussion with @vshampor it was decided to not spend time on fixing these tests as it could take big efforts.

Reply:
Thanks! I got it.